cluster-service/Makefile

```
@@ -37,14 +39,18 @@ deploy:
 	  -p IMAGE_REPOSITORY=app-sre/uhc-clusters-service \
 	  -p AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID=$${AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID} \
 	  -p FPA_CERT_NAME=${FPA_CERT_NAME} \
+	  -p ARM_HELPER_IDENTITY_CLIENT_ID=${ARM_HELPER_IDENTITY_CLIENT_ID} \
+	  -p ARM_HELPER_IDENTITY_CERT_NAME=${ARM_HELPER_IDENTITY_CERT_NAME}
 	  -p IMAGE_TAG=b16f630 | oc apply -f -
 
 deploy-integ:
 	AZURE_CS_MI_CLIENT_ID=$(shell az identity show \
 			-g ${RESOURCEGROUP} \
 			-n clusters-service \
 			--query clientId) && \
+	ARM_HELPER_IDENTITY_SERVICE_PRINCIPAL_CLIENT_ID=${ARM_HELPER_IDENTITY_CLIENT_ID} && \
 	oc process --local -f deploy/integration/cluster-service-namespace.yaml \
+	-p ARM_HELPER_IDENTITY_SERVICE_PRINCIPAL_CLIENT_ID=$${ARM_HELPER_IDENTITY_SERVICE_PRINCIPAL_CLIENT_ID} \
```

```suggestion
	-p ARM_HELPER_IDENTITY_CLIENT_ID=$${ARM_HELPER_CLIENT_ID} \
```
The `ARM_HELPER_IDENTITY_SERVICE_PRINCIPAL_CLIENT_ID` doesn't exist in the template.

---

renamed aptly